app/javascript/elm/tests/MainTests.elm

```diff
@@ -525,132 +528,173 @@ sessionWithTitle title =
     { session | title = title }
 
 
+selectedSession =
+    { title = "title"
+    , username = "username"
+    , sensorName = "sensor-name"
+    , average = 2.0
+    , min = 1.0
+    , max = 3.0
+    , timeRange = "time range"
+    , measurements = [ 1.0, 2.0, 3.0 ]
+    , id = 123
+    }
+
+
+selectedSessionWithId : Int -> SelectedSession
+selectedSessionWithId id =
+    { selectedSession | id = id }
+
+
 viewTests : Test
 viewTests =
     describe "view"
-        [ fuzz2 string string "session titles are displayed in the list" <|
+        [ fuzz2 string string "with no selection session titles are displayed in the list" <|
             \title1 title2 ->
-                { defaultModel | sessions = [ sessionWithTitle title1, sessionWithTitle title2 ], selectedSessionId = Nothing }
+                { defaultModel | sessions = [ sessionWithTitle title1, sessionWithTitle title2 ], selectedSession = NotAsked }
                     |> view
                     |> Query.fromHtml
                     |> Query.contains
                         [ Html.text title1
                         , Html.text title2
                         ]
-        , fuzz (intRange 1 10) "with modulo 50 sessions in the model the load more button is shown" <|
+        , fuzz (intRange 1 10) "with no selection and modulo 50 sessions in the model the load more button is shown" <|
             \times ->
-                { defaultModel | sessions = List.repeat (50 * times) session, selectedSessionId = Nothing }
+                { defaultModel | sessions = List.repeat (50 * times) session, selectedSession = NotAsked }
                     |> view
                     |> Query.fromHtml
                     |> Query.contains [ Html.text "Load More..." ]
-        , test "with 0 sessions in the model the load more button is not shown" <|
+        , test "with no selection and 0 sessions in the model the load more button is not shown" <|
```

Comment:
is this case is true for 0-49 sessions?

Reply:
I think so. But 0 is an edge case cause it's modulo 50 that's why I tested